filesystems/filesystems.tex

```diff
@@ -251,10 +251,9 @@ \subsection{Unix Directory Conventions}
   \item \keyword{~~} is the name of the home directory usually
 \end{enumerate}
 
-Note that \keyword{...} is NOT a valid representation of any directory.
-This not the grandparent directory.
-It \emph{could} however be the name of a file on disk.
-Though confusingly, \keyword{zsh} provides this as a handy shortcut to the grandparent directory should it exist.
+Unituitively, \keyword{...} does not refer to the `grandparent' directory (parent of the parent). More generally, there are only reserved aliases for the current directory and the parent directory.
```

Comment:
Typo

Unintuitively
There are -> they are

Reply:
I meant to write "there are", maybe the wording is just bad in general - will reword it